update-gomod.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
nit: maybe `update-cri-gomod.sh` or similar would be a slightly better name so it's apparent this just handles cri-related dependencies

Reply:
Okay. Will rename the script.